.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug Report 🐛
+about: Something isn't working as expected? Report it here.
```

Comment:
https://help.github.com/en/github/building-a-strong-community/manually-creating-a-single-issue-template-for-your-repository

Can you add labels for these too? Like question / bug / feature?

Reply:
Pushed!